Resources/Prototypes/_LostParadise/Traits/disabilities.yml

```
@@ -0,0 +1,18 @@
+      foodDelayMultiplier: 1.6
+      drinkDelayMultiplier: 1.6
+  requirements:
+    - !type:CharacterSpeciesRequirement
+      inverted: true
+      species:
+        - IPC
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - Borg
+        - MedicalBorg
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)